docs/getting-started/licensing/license-and-legal.md

```diff
@@ -1,27 +1,44 @@
 ---
 category: licensing
 meta-title: Editor license | CKEditor 5 Documentation
-meta-description: Choosing the right licence type for you needs.
+meta-description: Choosing the right license type for you needs.
 menu-title: Editor license
 order: 30
 modified_at: 2024-11-20
 ---
 
 # Editor license and legal terms
 
-The following legal notices apply to CKEditor&nbsp;5 and all software from CKEditor&nbsp;5 Ecosystem included with it.
+CKEditor is dual-licensed. This means we make it available under a choice between an open source and a commercial license.
```

Comment:
`under a choice` sounds a bit clunky, but I may be wrong, @godai78 could you check?

Reply:
FYI: This is straight from ckeditor.com new licensing page.

Reply:
Please open a ticket for ckeditor.com too.